projects/eval-pqmon/src/examples/basic/basic_example.c

```diff
@@ -225,6 +225,23 @@ int basic_pqm_firmware()
 	app_init_param.lwip_param.extra = NULL;
 #endif
 
+#if defined(PQM_CONN_ETH)
+	struct w5500_dev *mac_dev;
+	struct w5500_network_dev *net_dev;
+
+	status = w5500_init(&mac_dev, &w5500_ip);
```

Comment:
I think you can move this part in the `w5500_network_init()` function in order to simplify the application code.